### PR TITLE
Update dark mode toggle switch

### LINKED ISSUE #2497: Update dark mode toggle switch
The dark mode setting currently uses a toggle switch.

image

dcrdata has a nice icon for their dark mode.  We should use this icon as well.  It provides a better UX and its good for features to be consistent across the various Decred codebases.

image
image

### PR DESCRIPTION
**Pull request by vibros68**
close #2497 

This pull request update the dark mode toggle to be a new designed
toggle. The preview of the toggle here:
Screenshot from 2021-09-16 23-32-27

It also does some adjustments with nav header style follow [the comment](https://github.com/decred/politeiagui/issues/2497#issuecomment-890911752)

The preview behavior can be seen in the below video:

https://user-images.githubusercontent.com/84569563/133654875-0004d7f3-e322-4017-9b13-f40223b075f5.mp4

### CODE REVIEW

**Review by lukebp — CHANGES_REQUESTED** [review]
I'm seeig a slight delay from when I click the button to when the animation starts.  Is this delay intentional?


https://user-images.githubusercontent.com/9039877/133667819-09bf1cee-b377-47b1-94db-4c94da3cfc06.mp4

**Comment by vibros68** [comment]
@lukebp the current delay time for animation effect is 200 mili second. While the change of the variable is immediately. May be I should change the animation effect to 100, so the time seems to fit better

**Comment by lukebp** [comment]
What about taking out the delay entirely?  It doesn't look like there is one on test-proposals.


https://user-images.githubusercontent.com/9039877/133782184-d116c009-93bc-452d-9a40-f3a07d56c9b1.mp4

**Review by lukebp — APPROVED** [review]
tACK

Needs code approval from @tiagoalvesdulce.

**Review by amass01 — APPROVED** [review]
LGTM!

File: src/components/HeaderNav/HeaderNav.module.css [hunk 1]
```diff
@@ -1,9 +1,13 @@
-.navContainer {
+.sharedNavAtribute {
   height: 100%;
   display: flex;
   align-items: center;
 }
 
+.navContainer {
+  composes: sharedNavAtribute;
```
Comment (tiagoalvesdulce) [inline src/components/HeaderNav/HeaderNav.module.css:8]: good job using composes!

**Review by tiagoalvesdulce — APPROVED** [review]
LGTM!

**Pull request merged**